### PR TITLE
testing/py-html5lib: upgrade to 1.0.1

### PR DESCRIPTION
**Pull request by stwa**

### CODE REVIEW

File: testing/py-html5lib/APKBUILD [hunk 1]
```diff
@@ -1,41 +1,49 @@
...
 pkgdesc="A Python HTML parser"
 url="https://github.com/html5lib/html5lib-python"
 arch="noarch"
 license="MIT"
-depends="python2"
-depends_dev=""
-makedepends="python2-dev py-setuptools"
-install=""
-subpackages=""
+options="!check"
```
Comment (andypost) [inline testing/py-html5lib/APKBUILD]: there is test suite https://github.com/html5lib/html5lib-python#tests

File: testing/py-webencodings/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+# Maintainer: Stefan Wagner <stw@bit-strickerei.de>
+pkgname=py-webencodings
+_pkgname=webencodings
+pkgver=0.5.1
+pkgrel=0
+pkgdesc="Character encoding for the web"
+url="https://github.com/gsnedders/python-webencodings"
+arch="noarch"
+license="MIT"
+options="!check"
```
Comment (andypost) [inline testing/py-webencodings/APKBUILD]: there's tests as well https://github.com/gsnedders/python-webencodings/blob/master/tox.ini#L1

Comment (stwa) [reply testing/py-webencodings/APKBUILD]: I added the missing tests.

File: testing/py-html5lib/APKBUILD (hunk 1 above)
Comment (stwa) [reply testing/py-html5lib/APKBUILD]: Adding missing dependencies to run tests.

**Comment by stwa** [comment]
I did the requested changes, so this should be ready for another review.

**Comment by andypost** [comment]
Let's get @fabaff (maintainer) POV

**Comment by fabaff** [comment]
Fine with me :+1:

File: testing/py-webencodings/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+builddir="$srcdir"/python-$_pkgname-$pkgver
+
+build() {
+	cd "$builddir"
+	python2 setup.py build
+	python3 setup.py build
+}
+
+check() {
+	python2 setup.py test
```
Comment (andypost) [inline testing/py-webencodings/APKBUILD]: Needs add `cd "$builddir"`

File: testing/py-html5lib/APKBUILD [hunk 4]
```diff
@@ -1,41 +1,55 @@
...
+	cd "$builddir"
+	python2 setup.py build
+	python3 setup.py build
 }
 
-build() {
-	cd "$_builddir"
-	python2 setup.py build || return 1
+check() {
+	py.test-2
```
Comment (andypost) [inline testing/py-html5lib/APKBUILD]: needs add `cd "$builddir"`

Comment (andypost) [reply testing/py-html5lib/APKBUILD]: https://wiki.alpinelinux.org/wiki/APKBUILD_examples:Python#Multiversion_package_without_executables

File: testing/py-webencodings/APKBUILD (hunk 3 above)
Comment (stwa) [reply testing/py-webencodings/APKBUILD]: I had something in mind with newest abuild changes dirs to $builddir automatically.
Added.

File: testing/py-html5lib/APKBUILD (hunk 4 above)
Comment (stwa) [reply testing/py-html5lib/APKBUILD]: Added $builddir. Thanks for dropping in this link. I did some more changes according to the documentation.

**Pull request closed without merging**